src/command_logic.cc

```diff
@@ -761,4 +797,7 @@ initialize_command_logic() {
   CMD2_ANY_LIST("elapsed.greater", [](const auto&, const auto& args) {
     return apply_elapsed_greater(args);
   });
```

Comment:
return apply_elapsed_greater(args) at the end of args please add 0*1 or no of times to indicate the the change

Reply:
It looks like your PR already makes this change. If your PR gets merged in `master`, I can merge `master` back into my branch to bring in your changes, but trying to do that ahead of time will only cause problems.